app/assets/stylesheets/application.css.scss

```diff
@@ -14,29 +14,49 @@
  *= require_self
  */
 
- @import url('https://fonts.googleapis.com/css?family=Parisienne|Quicksand&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Parisienne|Quicksand&display=swap');
+
+// VARIABLES, MIXINS, ETC.
+
+$font-family-body: 'Verdana';
+$font-family-headers: 'Georgia';
+
+$alert-text-color: #8e0613;
+
+$button-bg-color: #ffffff75;
+$link-color: #13525f;
+$navbar-bg-color: #99D3DFBD;
+$card-bg-color: #99D3DF50;
+$card-border-color: #88BBD695;
+$container-bg-color: #E9E9E9F5;
+$container-border-color: #CDCDCD95;
```

Comment:
Nice, set the vars above, and then just reference them in the class selectors below.

I had tried setting these directly within class selectors once, and got an 'invalid CSS error'.

Note to team on this!   Best practices for styling, as all we have to do is reset one of these vars up top (vs having to record every single class selector)